corgi/core/models.py

```diff
@@ -255,7 +255,12 @@ def save_product_taxonomy(self):
         """update ('materialize') product taxonomy on all build components"""
         variant_ids = list(
             ProductComponentRelation.objects.filter(build_id=self.build_id)
-            .filter(type=ProductComponentRelation.Type.ERRATA)
+            .filter(
+                type__in=[
+                    ProductComponentRelation.Type.ERRATA,
+                    ProductComponentRelation.Type.CDN_REPO,
+                ]
```

Comment:
Small nit: Since this list isn't being changed anywhere, it would be better to make it a tuple:
```suggestion
                type__in=(
                    ProductComponentRelation.Type.ERRATA,
                    ProductComponentRelation.Type.CDN_REPO,
                )
```

Reply:
Ok, done. I also adjusted the stread_ids loop below to use a tuple.